Documentation/building/code-coverage.md

```diff
@@ -60,3 +60,18 @@ The results for this one library will then also show up in the aforementioned in
 And then once the run completes:
     
     ..\..\..\bin\tests\coverage\index.htm
+
+## Code coverage with mscorlib code
+
+Some code in the corefx contracts does not live in the corefx repo, e.g. System.Runtime, System.Text.Encoding. To run coverage reports for these projects, you need to build mscorlib locally from the coreclr repo.
+
+
+The following steps can be used manually to produce a coverage report, but a customizable batch file can be found [here](https://gist.github.com/hughbe/511331a3d264d48b390d1803475a91cf). Changing the parameters in the first couple of lines lets you run a coverage report easily for any mscorlib project.
+
+1. Build the local test project (`msbuild /T:Build`)
+3. Build coreclr locally in Debug (`build.cmd all Debug skiptests`)
+2. Navigate to the built test directory in the corefx bin (e.g. `bin/tests/AnyOS.AnyCPU.Debug/System.Runtime/dnxcore50` for `System.Runtime`
+4. Delete `coreclr.dll`, `mscorlib.dll` and `mscorlib.ni.dll` from that directory
```

Comment:
Note that it'll also work with release builds of coreclr/mscorlib.  The important things are that the symbols are available (PDBs are generated for release builds as well) and the coreclr/mscorlib bits match in terms of the build done and the source from which they were built.